builder/data/32bit/usr/bin/pwnlib

```diff
@@ -18,27 +38,27 @@ reload_brcm() {
     return 1
   fi
   sleep 2
-  iw dev wlan0 set power_save off
+  iw dev ${PWNY_WIFI_IFACE} set power_save off
   return 0
 }
 
```

Comment:
I was testing these config.txt changes for the iface on my rpizerow2, with and ac600 external antennae. (so realtek drivers). with this modification if you are using any antenna that is not running on the same driver as the onboard chipset it will turn the iface off, the check_brcm also seem susceptible to the same kind of issue as it relies on a driver name rather then iface. Ill try and look into a fix for this once I have time, ty for the mr to show all the places that is being set.

Reply:
```
export PWNY_WIFI_IFACE='wlan0'
export PWNY_MON_IFACE='wlan0mon'
export PWNY_WIFI_DRIVER=$(ls /sys/class/net/$PWNY_WIFI_IFACE/device/driver/module/drivers | cut -d":" -f2)

# Load configurations and adjust the monitor interface as needed
for conf in /usr/local/lib/python3.11/dist-packages/pwnagotchi/default.toml \
    /etc/pwnagotchi/default.toml \
    /etc/pwnagotchi/config.toml; do
    if grep -q main.iface $conf; then
        mondev=$(grep main.iface $conf | cut -d \" -f2)
        export PWNY_MON_IFACE=$mondev
    fi
done

# Function to check the status of the wireless interface
check_wireless_status() {
    local last_logs=$(journalctl -n 100 -k --since -5m | grep -Ei 'error|fail|critical')
    if [[ -z "$last_logs" ]]; then
        return 0
    else
        echo "Recent critical kernel logs detected:"
        echo "$last_logs"
        return 1
    fi
}

# Function to reload the wireless driver
reload_wireless_driver() {
    local driver_module=${1:-"$(ls /sys/class/net/$PWNY_WIFI_IFACE/device/driver/module/drivers | cut -d":" -f2)"}
    
    if ! modprobe -r $driver_module; then
        echo "Failed to unload driver $driver_module"
        return 1
    fi
    sleep 1
    
    if ! modprobe $driver_module; then
        echo "Failed to load driver $driver_module"
        return 1
    fi
    sleep 2
    
    iw dev $PWNY_WIFI_IFACE set power_save off
    return 0
}
```

Reply:
something like that may reload the correct drivers